### PR TITLE
[CSFS-153] Article Publishing Endpoints

### PR DESCRIPTION
**Pull request by rubyade**
Capture49

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,8 +0,0 @@
-# Mac DS_Store Files
```
Comment (adaugochi) [inline .gitignore:1]: You have deleted our `.gitignore` file that is inside the `Dufuna-Fem` folder. Kindly replace what you delete back

File: submissions/rubyAdeniregun/sparkPublishingHouse/.env.sample.txt [hunk 2]
```diff
@@ -0,0 +1,14 @@
+#Database config
```
Comment (adaugochi) [inline submissions/rubyAdeniregun/sparkPublishingHouse/.env.sample.txt:1]: This is not a `.txt` file. Kindly remove the `.txt` extension

**Review by adaugochi — CHANGES_REQUESTED** [review]
Well done @rubyade

**Review by adaugochi — CHANGES_REQUESTED** [review]
Your code is ok, but your documentation is not. Kindly message me privately let's have a call concerning it.

File: submissions/rubyAdeniregun/sparkPublishingHouse/app/controllers/ArticleController.php [hunk 3]
```diff
@@ -90,10 +92,9 @@ public function publish(Request $request, Response $response, $args) //An endpoi
         $data = $request->getParsedbody();
         $id = $args['id'];
         $article = Article::find($id);
-        $article->status = $data['status'];
+        $article->status = 'published';
```
Comment (Ireti203) [inline submissions/rubyAdeniregun/sparkPublishingHouse/app/controllers/ArticleController.php:95]: why are you not getting the status from the db?

**Review by Ireti203 — CHANGES_REQUESTED** [review]

**Review by Ireti203 — APPROVED** [review]

**Pull request merged**